integration-tests/src/test/kotlin/org/stellar/anchor/platform/suite/End2EndTests.kt

```diff
@@ -0,0 +1,6 @@
+package org.stellar.anchor.platform.suite
+
+import org.junit.platform.suite.api.SelectPackages
+import org.junit.platform.suite.api.Suite
+
+@Suite @SelectPackages("org.stellar.anchor.platform.e2e_test") class End2EndTests
```

Comment:
Let's follow the package naming conventions and call this `org.stellar.anchor.platform.e2etest`.